### ISSUE TITLE
Resolve circular dependency between org.eclipse.mylyn.monitor.* and org.eclipse.mylyn.context.*

### ISSUE THREAD

**Issue by ruspl-afed** [eclipse-mylyn/org.eclipse.mylyn.commons#16]
Currently [org.eclipse.mylyn.commons](https://github.com/eclipse-mylyn/org.eclipse.mylyn.commons) repository requires org.eclipse.mylyn.ide_feature feature to be built. We need to introduce an interface to resolve this.

**Comment by BeckerFrank** [comment]
Please se my comment in https://github.com/eclipse-mylyn/org.eclipse.mylyn.commons/issues/14#issuecomment-1328231133

The following classes in org.eclipse.commons need some parts from org.eclipse.mylyn.context

ActivityContextManager
AbstractUserInteractionMonitor

**Comment by ruspl-afed** [comment]
Of cause you are welcome to suggest your solution.

**Comment by ruspl-afed** [comment]
I'm not sure that this is the only circular dependency, let's postpone the implementation a bit until we can see the whole picture.

**Comment by BeckerFrank** [comment]
Sure!

**Comment by BeckerFrank** [comment]
I think that we no longer need this because we want to have a new big Repository with all plugins (https://github.com/eclipse-mylyn/.github/issues/15)
@ruspl-afed, @wimjongman: Should we close this Issue?
Thoughts?

**Comment by ruspl-afed** [comment]
Yes, we can postpone this issue . With one repository it will be easier to fix. 
Sorry for postponing the review of your changes @BeckerFrank - currently I'm focused to establish healthy releng.
